examples/plot_fraction_scan.py

```diff
@@ -54,10 +53,6 @@
     is_marker=True,
 )
 
-# Adding labels
-frac_plot.xlabel = "Test_x_label"
-frac_plot.ylabel = "Test_y_label"
-
 # Draw and save the plot
 frac_plot.draw()
 frac_plot.savefig("FractionScanPlot_test.png", transparent=False)
```

Comment:
no need to specify `transparent=False` anymore

Reply:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/46069353/169822236-85bf6c00-183c-4ed8-9f4b-fbceda25189d.png">

This was not updated in the docstring when the default value was changed to `False`. 
Could you please fix that @afroch?

Reply:
I would also use the dummy data generation for the example.

You could use this file e.g. https://gist.github.com/manuguth/0fe8d37d2cc78eb3dc25a502f1aa0af1